src/wallet/rpcdump.cpp

```diff
@@ -157,8 +157,11 @@ UniValue importprivkey(const JSONRPCRequest& request)
         if (!request.params[2].isNull())
```

Comment:
Nit: Update year in line 1? `// Copyright (c) 2009-2018 The Bitcoin Core developers`

Reply:
Done with a scripted diff